src/api/subscriptions.js

```diff
@@ -194,6 +200,12 @@ export const SubscriptionFlows = {
 };
 
 
+/** @enum {string} */
+export const LoginFlows = {
```

Comment:
Let's move to the `SubscriptionFlows` enum - they idea is to keep all flow names in one enum.

Reply:
done